### PR TITLE
Ng7 sidebar

### PR DESCRIPTION
**Pull request by daanishnasir**
ng7 sidebar with icons & keyboard navigation works. 

## Description

Component and examples are done for review. 

## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: angular/src/lib/sidebar/sidebar.component.html [hunk 1]
```diff
@@ -0,0 +1,141 @@
+
```
Comment (daanishnasir) [inline angular/src/lib/sidebar/sidebar.component.html]: Delete file

File: angular/src/lib/sidebar-body/examples/sidebar-body-default.component.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  styles: []
+})
+export class SideBarBodyDefaultComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit() {
+  }
+
+}
```
Comment (mathsusan) [inline angular/src/lib/sidebar-body/examples/sidebar-body-default.component.ts]: This just looks like the default code .. guessing you'll clean it up

File: angular/src/lib/sidebar-nav/sidebar-nav.component.ts [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  // Need descendants true to include nested nav items
+  @ContentChildren(SidebarNavItemComponent, {descendants: true}) navItemLists: QueryList<SidebarNavItemComponent>;
+
+  public navItems: SidebarNavItemComponent[] = [];
+
+  constructor(  ) { }
+
+  ngAfterContentInit() {
+    this.navItems = this.navItemLists.toArray();
+    console.log('content sidebar-nav items: ', this.navItems);
```
Comment (daanishnasir) [inline angular/src/lib/sidebar-nav/sidebar-nav.component.ts]: removed console logs

File: react/src/app/containers/Playground/insertExampleHere.js [hunk 4]
```diff
@@ -1,16 +1,155 @@
 import React from 'react';
```
Comment (daanishnasir) [inline react/src/app/containers/Playground/insertExampleHere.js]: will revert back to master playground file

**Pull request merged**